composer.json

```diff
@@ -27,6 +28,7 @@
         "symfony/phpunit-bridge": "^2.7 || ^3.0"
     },
     "provide": {
+        "ext-mongo": "*",
```

Comment:
Yes, but this does not explain why this provide key.

The fact `alcaeus/mongo-php-adapter` provides `ext-mongo`, why not.

But `sonata-project/doctrine-mongodb-admin-bundle`, certainly not.